samples/subsys/usb/dfu/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
 project(NONE)
 
+set(DTC_OVERLAY_FILE "${CMAKE_CURRENT_SOURCE_DIR}/dts.overlay")
```

Comment:
You don't need this anymore, you only need to enable `CONFIG_BOOTLOADER_MCUBOOT` in the `.conf`